app/scraper.py

```diff
@@ -3,12 +3,22 @@
 import requests
 from bs4 import BeautifulSoup
 
+try:
+    from imp import reload
```

Comment:
Instead of `from imp import reload` , `reload` would work. Please read more about it [here](https://docs.python.org/2/library/functions.html#reload)

Reply:
@vaibhavsingh97 importlib is for python3 compatibility

Reply:
in try block you are using reload for python2  so in python2 relaod is already present. Please read the reference link i shared with you.